### PR TITLE
Minor changes in convention.

### PR DESCRIPTION
**Pull request by samdevsung**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -4,8 +4,8 @@
     * Prefer descriptive names over short form
     * Follow Coding Convention
     * The class name should be a noun
-    * Avoid clutters like _l, m_, o_,O1, Ol, lO, etc
-    * Avoid using non-ASCII characters and words from the local language
+    * Avoid ambiguious names like _l, m_, o_,O1, Ol, lO, etc
```
Comment (wizzenalum) [inline readme.md:7]: i think clutter name can work fine, is there any benefit of doing it.

File: readme.md [hunk 2]
```diff
@@ -4,8 +4,8 @@
...
+    * Avoid using non-ASCII characters and characters from the local language
```
Comment (wizzenalum) [inline readme.md:8]: can you remove one 'characters'.

**Review by wizzenalum — CHANGES_REQUESTED** [review]
make a appropriate name of the pull request

File: readme.md (hunk 1 above)
Comment (samdevsung) [reply readme.md:7]: nope clutter doesn't convey your actual meaning of using ambiguous characters.
the spelling of ambiguous is wrong.

**Pull request merged**